include/database.py

```diff
@@ -59,3 +59,69 @@ def insert_returning(connectionString, sql, parameters):
     except psycopg2.DatabaseError as err:
         sys.stderr.write(f'A general database error occurred: {err}')
         raise DatabaseError
+
+
+def remove_record(connectionString, sql, parameters):
+    try:
+        # Open the connection, override any timeout provided with something shorter
+        # since we expect to be running interactively
+        connection = psycopg2.connect(connectionString, connect_timeout=1)
+        cursor = connection.cursor()
+
+        # Execute the query, note the rows
+        cursor.execute(sql, (parameters['id'],))
+
+        # Clean-up and return
+        connection.commit()
+        # check if the study exists
+        if cursor.rowcount:
+            print("Study deleted successfully")
+        else:
+            print("Study does not exist")
+
+        cursor.close()
+
+    except psycopg2.OperationalError as err:
+        sys.stderr.write(f'An error occurred connecting to the database: {err}')
+        raise DatabaseError
+
+    except psycopg2.DatabaseError as err:
+        sys.stderr.write(f'A general database error occurred: {err}')
+        raise DatabaseError
+
+    except psycopg2.Error as e:
```

Comment:
Typically you should avoid since character variable names (loop iterators are an exception, but generally a manual of style will dictate this)